package.json

```diff
@@ -111,10 +111,10 @@
     "@babel/preset-env": "^7.13.9",
     "@babel/preset-react": "^7.12.13",
     "@babel/preset-typescript": "^7.13.0",
-    "@storybook/addon-actions": "^6.2.5",
-    "@storybook/addon-essentials": "^6.2.5",
-    "@storybook/addon-links": "^6.2.5",
-    "@storybook/react": "^6.2.5",
+    "@storybook/addon-actions": "^6.2.9",
+    "@storybook/addon-essentials": "^6.2.9",
+    "@storybook/addon-links": "^6.2.9",
+    "@storybook/react": "^6.2.9",
```

Comment:
Did these need upgrading for weback 5?

Reply:
Good call out, storybook isn't working with webpack 5. I am going to open an issue in the storybook project to get it working.

Reply:
Got it fixed by following this suggestion https://github.com/storybookjs/storybook/issues/14497#issuecomment-818723376